airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py

```diff
@@ -52,6 +54,26 @@ async def _run(self) -> StepResult:
             return StepResult(self, status=StepStatus.SUCCESS, stdout=f"No manifest found for {self.context.docker_image}.")
 
 
+class CheckPypiPackageDoesNotExist(Step):
```

Comment:
I'm wondering if this Step might be re-used  and would be better placed in the shared module

Reply:
I actually like it being localized for now - those "generic" places can get crowded quickly. What about leaving it in there for now and moving it out once we need it?